tests/json/bitcoin-config-test.json

```diff
@@ -0,0 +1,11 @@
+{
+  "bitcoinPeerUri": "unused",
+  "bitcoinWalletImportString": "cVneRge89Va3mrxv7dehTYTknbgLVan8rY5hkbXUwdwXtZrrueGY",
```

Comment:
And all of the scanners running over github for keys go wild... I'd generally generate a new key during testing, but this is fine 🔥🔥🔥